code/mobile/android/PhoneVR/app/src/main/cpp/alvr_main.cpp

```diff
@@ -256,6 +447,10 @@ extern "C" JNIEXPORT void JNICALL Java_viritualisres_phonevr_ALVRActivity_sendBa
 extern "C" JNIEXPORT void JNICALL Java_viritualisres_phonevr_ALVRActivity_renderNative(JNIEnv *,
                                                                                        jobject) {
     try {
+        if (CTX.eglContext == nullptr) {
```

Comment:
this is not necessary, it's a leftover from my earlier attempts at getting this to work